misc/video_types.py

```
@@ -5,101 +5,205 @@
+        return None
+
+    try:
+        video_file = BufferedInputFile(video_data, filename=f"{video_id}.mp4")
+        message = await bot.send_video(
+            chat_id=STORAGE_CHANNEL_ID,
+            video=video_file,
+            disable_notification=True,
+        )
+        if message.video:
+            return message.video.file_id
+        return None
```

Ruff (0.14.10)

Consider moving this statement to an `else` block

(TRY300)